### ISSUE TITLE
Bad things happen with sessions?

### ISSUE THREAD

**Issue by robbrandt** [zikula/core#3614]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.5.dev
| PHP Version       | 5.6.10

### Expected behavior
Pages should load.

### Actual behavior
Pages don't load.  Any page so far.  Cursor just spins, eventually times out, no error. Info below describes trying to load the home page, not anything that would require a login.

### Steps to reproduce

Not sure at this point.  I was logged in as an admin, doing repeated installs/uninstalls for a MOST module I am developing.  However this doesn't seem to be MOST related.  Has my admin session expired or corrupted?

I started up the debugger to see where the fail is happening, amd I've traced it so far to 
SessionHandlerProxy.php, line 67-70
public function read($sessionId)
    {
        return (string) $this->handler->read($sessionId);
    }
where sessionid is 68e3663f45a36f112f7fa76279d7470b and 
handler is Symfony\Component\HttpFoundation\Session\Storage\Handler\NativeFileSessionHandler, value [0].

It gets there via:
Core.php line 501-503
if (ServiceUtil::getManager()->getParameter('installed')) {
                ModUtil::load('ZikulaSecurityCenterModule');
            }
and UserUtil.php 756-758
if ($uid == -1) {
            $uid = SessionUtil::getVar('uid', null);
        }

I got around the issue by deleting the 68e3663f45a36f112f7fa76279d7470b file in app/cache/prod/sessions, but this isn't a solution for a live web site.

**Comment by craigh** [comment]
a ral web site wouldn't be installing and uninstalling and developing ...

you can delete the entire cache/prod directory. it will rebuild itself.

you should set `debug:true` when developing and/or `env:dev`

I'm guessing this is a "one off" behavior due to your dev work. If you can come up with a reproducible set of steps with a "fully formed" module, please let me know.

**Comment by robbrandt** [comment]
It's possible.  The only odd thing I can think of that I did was skipping a page refresh after uninstalling and before installing again.  Specifically:

* uninstall module (notice displayed: Module Uninstalled)
* Delete entire module directory
* Copy in new module directory
* When clicking the install module icon, notice displayed is still Module Uninstalled.

It was during this installation that the site became unresponsive.  This could confuse the system because (I think?) each delete/copy of the module files gets a new module ID number.  So I deleted ID 24, and after deleting/copying the new ID would be 25.  But without the page refresh it was still trying to install 24.
